{{cookiecutter.project_name}}/{{cookiecutter.app_name}}/api/views.py

```diff
@@ -1,18 +1,17 @@
 from flask import Blueprint, current_app, jsonify
 from flask_restful import Api
 from marshmallow import ValidationError
-
 from {{cookiecutter.app_name}}.extensions import apispec
 from {{cookiecutter.app_name}}.api.resources import UserResource, UserList
-from {{cookiecutter.app_name}}.api.resources.user import UserSchema
+from {{cookiecutter.app_name}}.api.schemas.user import UserSchema
```

Comment:
Since you're re-exporting `UserSchema` in `api/schemas/__init__.py` you should be able to reduce this import to `from {{cookiecutter.app_name}}.api.schemas import UserSchema`

Reply:
And this...